analyses/gene-set-enrichment-analysis/02-model-gsea.Rmd

```diff
@@ -233,14 +233,14 @@ for(i in 1:length(rna_library_list)){
     pull(harmonized_diagnosis) %>% 
     unique() %>% length()
   
-  if(harmonized_diagnosis_n>=2){
+  #if(harmonized_diagnosis_n>=2){
```

Comment:
@sangeetashukla, Why are you removing >2 cutoffs?

Reply:
@ewafula The issue description requires "Check to confirm that ... all harmonized diagnosis ANOVA statistics **don't meet** the >2 cutoffs". Am I missing something? On the other hand, as @jharenza suggests, I can go ahead and remove harm_dx altogether, keeping broad_hist and cancer_group.